src/layout/Header/ui/Header.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC } from "react";
+import { useTranslation } from "shared/hooks/i18nHook";
+
+export const Header: FC = () => {
+
+    const { t } = useTranslation('main');
```

Comment:
Let's rename main.json file to common.json file, because main.json file is related to Main page and the idea is to call files with translations based on the page where it's used Eg for Main page we'll use main.json file and this file will be loaded only when Main page is loaded and used, if user doesn't open this page file with translations also won't be loaded